app/src/main/res/layout/activity_offer.xml

```diff
@@ -21,69 +22,18 @@
         android:id="@+id/offerToolbar"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        app:navigationIcon="@drawable/ic_back"
         tools:layout_height="?attr/actionBarSize">
 
-        <LinearLayout
-            android:layout_width="match_parent"
+        <TextView
```

Comment:
Should this be an `ImageView` of the Hedvig Wordmark? 🤔

Reply:
Yeah I thought about that as well, do we have that as a resource already or should I export?

Reply:
Found it :)